src/inspector/view/qml/MuseScore/Inspector/general/playback/internal/HairpinsExpandableBlank.qml

```diff
@@ -79,17 +79,6 @@ ExpandableBlank {
             minValue: 0
         }
 
-        CheckBoxPropertyView {
-            id: singleNoteCheckBox
```

Comment:
This id is still used in velocityChangeTypeSection

`navigationRowStart: singleNoteCheckBox.navigation.row + 1`

Reply:
Got it: replaced the id with the one directly above it. I assume this fixes it, but is there a difference between using `.navigation.row` and `navigationRowEnd`?

Reply:
You should use navigationRowEnd here:

navigationRowStart:  velocityChangeSection.navigationRowEnd + 1

Reply:
>is there a difference between using .navigation.row and navigationRowEnd?

Some controls in Inspector may occupy a range of navigation indexes. navigationRowEnd is needed to indicate the end of these occupied indexes

Reply:
Thanks, makes sense! Just ammended the commit with `navigationRowEnd`.